website/server/controllers/api-v3/tasks.js

```diff
@@ -341,6 +234,14 @@ api.updateTask = {
     // see https://github.com/Automattic/mongoose/issues/2749
 
     let savedTask = await task.save();
+
+    if (task.group.id && task.assignedUsers.length > 0) {
+      let group = await Group.getGroup({user, groupId: task.group.id, populateLeader: false});
```

Comment:
you should limit the fields fetched here or at least exclude the chat because it's very large